### DISCUSSION TITLE
Support for mapping MIDI keys

### DISCUSSION THREAD

**Discussion by aleksandr-vin** [pqrs-org/Karabiner-Elements#3763] [Ideas]
Hi guys,

How do you look on the idea to add MIDI keys support to the tool? Maybe not fully duplex like mapping from and to MIDI keys, but starting from using them in [from event definition](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/from/)?

**Comment by ELLIOTTCABLE** [comment]
I'd also love to see MIDI support; at the moment, even though I already use Karabiner on all my machines for unrelated keymapping, I have to install a separate, additional tool to map MIDI to keypresses or vice-versa. It would be so neat to have that rolled directly into Karabiner!
